lib/Parser/rterrors.h

```diff
@@ -275,6 +275,8 @@ RT_ERROR_MSG(JSERR_DeletePropertyWithSuper, 5146, "Unable to delete property '%s
 RT_ERROR_MSG(JSERR_DetachedTypedArray, 5147, "%s: The ArrayBuffer is detached.", "The ArrayBuffer is detached.", kjstTypeError, 0)
 RT_ERROR_MSG(JSERR_AsmJsCompileError, 5148, "%s: Compiling asm.js failed.", "Compiling asm.js failed.", kjstError, 0)
 
+RT_ERROR_MSG(JSERR_ImmutablePrototypeSlot, 5149, "%s: Can't set the prototype slot of this object.", "Can't set the prototype slot of this object.", kjstTypeError, 0)
```

Comment:
I wouldn't mention _slot_ here since users don't need to know the language internals. I think it would be better to say `Can't set the prototype of this object` instead.